src/components/map/toolbar/EventLog.tsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
 import { observer } from 'mobx-react';
-import { FaTimes, FaExclamation } from 'react-icons/fa';
+import classnames from 'classnames';
+import { FaTimes, FaExclamation, FaUndo } from 'react-icons/fa';
 import { FiClipboard } from 'react-icons/fi';
+import ILogEntry from '~/models/ILogEntry';
+import logActions from '~/enums/logActions';
+import entityNames from '~/enums/entityNames';
```

Comment:
Because the enum is a discription of the name of the entity, such as route, routePath or likewise. EntityName is a common term used to declare these things. The term has almost 1 million hits on google, compared to 400.000 for terms such as "entityType". https://www.google.fi/search?ei=iKX7W8zkCMOTsAGX5JjoDg&q=%22entityname%22&oq=%22entityname%22

Reply:
"routePath", "route", etc refer to the Models in Jore. They are all models whose type/name is IRoutePath, IRoute. So maybe both type/name can be used...